apps/nerves_hub_web_core/lib/nerves_hub_web_core/devices/update_available.ex

```diff
@@ -0,0 +1,48 @@
+defmodule NervesHubWebCore.Devices.UpdateAvailable do
```

Comment:
```suggestion
defmodule NervesHubWebCore.Devices.UpdatePayload do
```
I would probably call this `UpdatePayload` since it is more than the context of "update available"